lib/timescaledb/rails/extensions/active_record/schema_statements.rb

```diff
@@ -116,8 +116,15 @@ def remove_hypertable_reorder_policy(table_name, _index_name = nil)
         #     'temperature_events', "SELECT * FROM events where event_type = 'temperature'"
         #   )
         #
-        def create_continuous_aggregate(view_name, view_query)
-          execute "CREATE MATERIALIZED VIEW #{view_name} WITH (timescaledb.continuous) AS #{view_query};"
+        def create_continuous_aggregate(view_name, view_query, force: false)
+          if force
+            execute "DROP MATERIALIZED VIEW #{quote_table_name(view_name)} CASCADE;" if view_exists? view_name
```

Comment:
allows for re-running the same migration idempotently

Reply:
i think it's similar to the `force:` parameter to `create_table`.
I added this and set it to true because we're creating multiple CAs in one migration, and if it fails halfway through, you can't re-run it, since you can't create a materialized view that already exists

Reply:
Ahhh, I forgot the `disable_ddl_transaction!` for continuous aggregates, the re-run part makes sense now, thanks for explaining it 👍